Sources/Subs-BoardIndex.php

```diff
@@ -296,6 +297,26 @@ function getBoardIndex($boardIndexOptions)
 					$row_boards[$row_board['id_parent']]['num_posts'] += $row_board['num_posts'];
 					$row_boards[$row_board['id_parent']]['num_topics'] += $row_board['num_topics'];
 				}
+
+				if (forum_time(true, $row_boards[$row_board['id_parent']]['poster_time']) < forum_time(true, $row_board['poster_time']))
```

Comment:
I don't think you need to call `forum_time()` here. Just use the raw poster_time values.